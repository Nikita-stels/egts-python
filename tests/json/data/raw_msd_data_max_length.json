{
  "transport": {
    "pt": 1
  },
  "service": {
    "sdr": [
      {
        "rfl": {
          "ssod": 0,
          "rsod": 0
        },
        "tm": "0x0fd90000",
        "sst": 10,
        "rst": 10,
        "rd": [
          {
            "srt": 40,
            "srd": {
              "fm": 0,
              "msd": "0x5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F454E4421"
            }
          }
        ]
      }
    ]
  }
}
