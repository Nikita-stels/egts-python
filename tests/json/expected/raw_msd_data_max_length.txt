EGTS protocol validator v.1.0  Copyright JSC "Navigation-information systems", 2011
------------------------------------------------------------------------------------
File: 'D:\PycharmProjects\egts_protocol\tests\tmp_msg.bin'
Packet data: 
 0100000B0083000100019478000100040000D90F0A0A287500005F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F454E44213D9D


EGTS Transport Layer:
---------------------
  Validating result   - 0 (OK)

  Protocol Version    - 1
  Security Key ID     - 0
  Flags               - 00000000b (0x00)
       Prefix         - 00
       Route          -   0
       Encryption Alg -    00
       Compression    -      0
       Priority       -       00 (the highest)
  Header Length       - 11
  Header Encoding     - 0
  Frame Data Length   - 131
  Packet ID           - 1
  No route info       -
  Header Check Sum    - 0x94

EGTS Service Layer:
---------------------
  Validating result   - 0 (OK)

  Packet Type         - EGTS_PT_APPDATA
  Service Layer CS    - 0x9D3D

    Service Layer Record:
    ---------------------
    Validating Result    - 0 (OK)

    Record Length               - 120
    Record Number               - 1
    Record flags                -     00000100b (0x04)
        Sourse Service On Device    - 0
        Recipient Service On Device -  0
        Group Flag                  -   0
        Record Processing Priority  -    00 (the highest)
        Time Field Exists           -      1
        Event ID Field Exists       -       0
        Object ID Field Exists      -        0
    Time                        - 0x0FD90000 (05.06.2018 07:25:52 GMT)
    Source Service Type         - 10 (EGTS_ECALL_SERVICE) 
    Recipient Service Type      - 10 (EGTS_ECALL_SERVICE) 

       Subrecord Data:
       ------------------
       Validating Result   - 0 (OK)

       Subrecord Type      - 40 (EGTS_SR_RAW_MSD_DATA)
       Subrecord Length    - 117
       Format                 - 0
       Raw MSD data           - 116 bytes [5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F5F564552595F4C4F4E475F444154415F454E4421]


-------------------------------------------------------------
Press any key to exit...
